src/mesh/raspihttp/PiWebServer.cpp

```diff
@@ -0,0 +1,527 @@
+        o_free(url_dup_save);
+        free(real_path); // realpath uses malloc
+        return U_CALLBACK_CONTINUE;
+    } else {
+        LOG_DEBUG("Static File Server - Error, user_data is NULL or inconsistent\n");
+        return U_CALLBACK_ERROR;
+    }
+}
+
+static void handleWebResponse() {}
+
+/*
+ * Adapt the radioapi to the Webservice handleAPIv1ToRadio
+ * Trigger : WebGui(SAVE)->WebServcice->phoneApi
+ */
+int handleAPIv1ToRadio(const struct _u_request *req, struct _u_response *res, void *user_data)
+{
+    LOG_DEBUG("handleAPIv1ToRadio web -> radio  \n");
+
+    ulfius_add_header_to_response(res, "Content-Type", "application/x-protobuf");
+    ulfius_add_header_to_response(res, "Access-Control-Allow-Headers", "Content-Type");
+    ulfius_add_header_to_response(res, "Access-Control-Allow-Origin", "*");
+    ulfius_add_header_to_response(res, "Access-Control-Allow-Methods", "PUT, OPTIONS");
+    ulfius_add_header_to_response(res, "X-Protobuf-Schema",
+                                  "https://raw.githubusercontent.com/meshtastic/protobufs/master/mesh.proto");
+
+    if (req->http_verb == "OPTIONS") {
+        ulfius_set_response_properties(res, U_OPT_STATUS, 204);
+        return U_CALLBACK_CONTINUE;
+    }
+
+    byte buffer[MAX_TO_FROM_RADIO_SIZE];
+    size_t s = req->binary_body_length;
+
+    memcpy(buffer, req->binary_body, MAX_TO_FROM_RADIO_SIZE);
+
+    // FIXME* Problem with portdunio loosing mountpoint maybe because of running in a real sep. thread
+
+    portduinoVFS->mountpoint("/home/marc/.portduino/default");
+
+    LOG_DEBUG("Received %d bytes from PUT request\n", s);
+    webAPI.handleToRadio(buffer, s);
+    LOG_DEBUG("end web->radio  \n");
+    return U_CALLBACK_COMPLETE;
+}
+
+/*
+ * Adapt the radioapi to the Webservice handleAPIv1FromRadio
+ * Trigger : WebGui(POLL)->handleAPIv1FromRadio->phoneapi->Meshtastic(Radio) events
+ */
+int handleAPIv1FromRadio(const struct _u_request *req, struct _u_response *res, void *user_data)
+{
+
+    LOG_DEBUG("handleAPIv1FromRadio radio -> web\n");
+    std::string valueAll;
+
+    // Status code is 200 OK by default.
+    ulfius_add_header_to_response(res, "Content-Type", "application/x-protobuf");
+    ulfius_add_header_to_response(res, "Access-Control-Allow-Origin", "*");
+    ulfius_add_header_to_response(res, "Access-Control-Allow-Methods", "GET");
+    ulfius_add_header_to_response(res, "X-Protobuf-Schema",
+                                  "https://raw.githubusercontent.com/meshtastic/protobufs/master/mesh.proto");
+
+    uint8_t txBuf[MAX_STREAM_BUF_SIZE];
+    uint32_t len = 1;
+
+    if (valueAll == "true") {
+        while (len) {
+            len = webAPI.getFromRadio(txBuf);
+            ulfius_set_response_properties(res, U_OPT_STATUS, 200, U_OPT_BINARY_BODY, txBuf, len);
+            const char *tmpa = (const char *)txBuf;
+            ulfius_set_string_body_response(res, 200, tmpa);
+            LOG_DEBUG("\n----webAPI response all:----\n");
```

Comment:
I suggest to remove/comment out or wrap into another #ifdef these logs in this methos. They are totally spamming the debug log with nine entries every 2 seconds!